release-notes.hbs.md

```diff
@@ -37,6 +37,15 @@ This release includes the following changes, listed by component and area.
 
 - Feature description.
 
+#### <a id='1-7-0-api-autoreg'></a> v1.7.0 Features: API Auto Registration
```

Comment:
Added release notes here. Check if you would like to edit anything. @LittleBaiBai @csterwa @karinepires